### PR TITLE
textadept: 9.3 -> 10.2

### PR DESCRIPTION
**Pull request by brainrake**
###### Motivation for this change
Package was out of date.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/editors/textadept/deps.nix [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{
+    "542782a4df7d.zip" = {
```
Comment (Mic92) [inline pkgs/applications/editors/textadept/deps.nix]: Was this file automatically generated?

**Comment by 7c6f434c** [comment]
Mention me if you want to do a PR sharing the `deps.nix` generator somewhere near the package (in case it was an external script)

File: pkgs/applications/editors/textadept/deps.nix (hunk 1 above)
Comment (brainrape) [reply pkgs/applications/editors/textadept/deps.nix]: no, but i put it in a format where it could easily be

Comment (brainrape) [reply pkgs/applications/editors/textadept/deps.nix]: i would have committed the script if there was one ofc

**Comment by brainrape** [comment]
There's no script, I just like to imagine I am one. And now it's easier to write, and more probable that someone will. 
Unfortunately the source for the data is a makefile with some ad-hoc stuff, and one of the urls points to a hg `tip` (like a git branch, content may change). I used the latest revision at the time of packaging, seems to work.

**Comment by 7c6f434c** [comment]
Sure, and the list is also convenient to use.

**Pull request merged**